app/account/forms.py

```diff
@@ -151,4 +152,15 @@ class EditProfileForm(Form):
         description="https://",
         validators=[URL(), Optional()]
     )
+    affiliations = SelectMultipleField(
+        'Affiliations',
+        default=[]
+    )
     submit = SubmitField('Update profile')
+
+    def __init__(self, *args):
```

Comment:
I'm overriding the constructor so that we can initialize the affiliation `choices` using the actual `AffiliationTag`s available (instead of hard-coding them in).